wiki/Gameplay/Daily_challenge/en.md

```diff
@@ -0,0 +1,63 @@
+# Daily Challenge
+
+The **daily challenge** is a multiplayer mode in osu!(lazer) where players are able to build up a *daily* streak by passing beatmaps consecutively with increasing difficulty.
+
+Each day introduces one new map from the [Featured Artist](/wiki/People/Featured_Artists) listing with increased difficulty. Once every 7 days however, the difficulty resets again.
+
+Sometimes, the maps come with a mod forced onto users, which requires them to pass the map with the selected mod.
+
+## Game menu
+
+From the main menu, the daily challenge menu can be accessed with the following steps:
+
+1. Click the `Play` button or press `P`.
+2. Click the `Daily challenge` button or press `D`.
+
+![](img/Daily-challenge-menu.png "Screenshot of the daily challenge menu during November 7, 2024.") 
+
+On entering, the user is presented with an intro showing the beatmap the user has to pass, and the mods they have to pass it with.
+
+After the intro ends, the user can view the data for today's challenge. The left side displays various score-related information like the total pass count and the cumulative total score. In the middle, a leaderboard shows the top scores achieved by players. Users are able to discuss the daily challenge in the chat on the right.
+
+## Beatmap difficulty range
+
+![](img/Beatmap-selection-criteria.png "Specific star ratings beatmaps have to follow for the entire week")
+
+## Streak milestones
+
+Play streaks are displayed on the user profile with different colours depending on certain streak milestones.
+
+#### Rank breakdown 
+
+| Rank  | Total streak | Daily streak | Weekly streak |
+| :-: | :-: | :-: | :-: |
+| Lustrous  | 1081 days | 360 days | 53 weeks |
+| Radiant | 721 days | 240 days | 36 weeks |
+| Rhodium | 361 days| 120 days | 19 weeks |
+| Platinum | 181 days | 60 days | 10 weeks |
+| Gold | 91 days | 30 days | 6 weeks |
+| Silver | 31 days | 10 days | 3 weeks  |
+| Bronze | 16 days | 5 days | 2 weeks |
+| Iron | less than 16 days | less than 5 days | less than 2 weeks |--->
```

Comment:
```suggestion
| Iron | less than 16 days | less than 5 days | less than 2 weeks |
```

Reply:
must have left that after deleting the comment, oops, my bad